classes/order/OrderDetail.php

```diff
@@ -755,7 +755,7 @@ public function createList(Order $order, Cart $cart, $id_order_state, $product_l
 
         unset(
             $this->vat_address,
-            $products, $this->customer
```

Comment:
I see that `OrderDetail::createList()` is used in PaymentModule.
Modifying its unspecified behavior is highly dangerous.
May I suggest some other ideas ?
- create a new function instead of using/modifying `createList()`
- add a flag (yes I know 😱) to modify this one

I am very scared of this change

Reply:
Please check again... The variable isn't used in  the method and it's the alone reference. :wink:

Reply:
You're right, I was confused by this stupid code 😓